src/AdminApi/ScriptTag/Builders/GetScriptTagsRequest.php

```diff
@@ -0,0 +1,269 @@
+<?php
+
+namespace Yaspa\AdminApi\ScriptTag\Builders;
+
+use DateTime;
+use Yaspa\Constants\RequestBuilder;
+use Yaspa\Interfaces\PagingRequestBuilderInterface;
+use Yaspa\Traits\AuthorizedRequestBuilderTrait;
+
+/**
+ * Class GetScriptTagsRequest
+ *
+ * @package Yaspa\AdminApi\ScriptTags\Builders
+ * @see https://help.shopify.com/api/reference/scripttag#index
+ */
+class GetScriptTagsRequest implements PagingRequestBuilderInterface
+{
+    use AuthorizedRequestBuilderTrait;
+
+    const URI_TEMPLATE = 'https://%s.myshopify.com/admin/script_tags.json';
+
+    /** @var array|int[] $ids */
+    protected $ids;
```

Comment:
`ids` doesn't appear in list of fields in https://help.shopify.com/api/reference/scripttag#index